src/verifier.cc

```diff
@@ -480,6 +480,32 @@ bool IsTexelEqualToExpected(const std::vector<double>& texel,
   return true;
 }
 
+std::vector<double> GetSortedTexelInOrderOfRGBA(
```

Comment:
GetTexelInRGBA, sorted doesn't really mean anything in this context I don't think.

Reply:
done